samples/cellular/modem_shell/src/sock/sock.c

```diff
@@ -1202,7 +1202,7 @@ int sock_close(int socket_id)
 
 static int sock_rai_option_set(int fd, int option, char *option_string)
 {
-	int err = setsockopt(fd, SOL_SOCKET, option, NULL, 0);
+	int err = setsockopt(fd, SOL_SOCKET, SO_RAI, &option, sizeof(option));
 
 	if (err) {
 		mosh_error("setsockopt() for %s failed with error %d", option_string, errno);
```

Comment:
There is similar print four lines below.

```suggestion
		mosh_error("setsockopt() for SO_RAI with value %s failed with error %d", option_string, errno);
```